admin_manual/installation/source_installation.rst

```diff
@@ -2,47 +2,177 @@
+     (make sure to thank them - or find a way to help them out - if you get the opportunity!) and partner Hansson
+     IT.
+   - Helps you create a personal or corporate Nextcloud Server faster and easier. 
+   - Can be used install directly on a clean Ubuntu Server (as an install script) or downloaded as a fully 
+     functioning VM. Full feature list available `here 
+     <https://github.com/nextcloud/vm/?tab=readme-ov-file#server-installation-simplified-cloud>`_.
+   - `Official Documentation <https://github.com/nextcloud/vm/>`_ (**Note**: see Official Documentation 
+     for VM specific installation and usage details).
+* NextcloudPi (aka: NCP)
+   - A community-driven installation method that is actively maintained by amazing Nextcloud community members 
+     (make sure to thank them - or find a way help them out - if you get the opportunity!).
+   - A ready to use image for Virtual Machines, Raspberry Pi, Odroid HC1, Rock64 and other boards. Also supports
+     LXD and LXC containers and there is an install script for the latest supported Debian based system as well.
+   - It will setup everything for you and include scripts for automated installation of apps like: Collabora, 
+     OnlyOffice, Talk and so on. Full feature list available 
+     `here <https://github.com/nextcloud/nextcloudpi?tab=readme-ov-file#features>`_.
+   - `Official Documentation <https://github.com/nextcloud/nextcloudpi>`_ (**Note**: see Official Documentation 
+     for NCP specific installation and usage details).
+* Nextcloud Community Docker Image
+   - A community-driven installation method that is actively maintained by amazing Nextcloud community members 
+     (make sure to thank them - or find a way help them out - if you get the opportunity!).
+   - Two editions of the image are provided for building different types of Nextcloud stacks: the Apache image 
+     contains a full Nextcloud installation including an Apache web server. The second option is an FPM 
+     installation and runs a FastCGI process that serves your Nextcloud installation. The standard images are
+     Debian-based, but Alpine variants are offered as well.
+   - This image is designed for expert use and intended to be used in a micro-service environment where you can
+     incorporate as a building block in your own Nextcloud stack (i.e. you will not need to install/update 
+     Nextcloud, but you will need to supply your preferred web, reverse proxy, HTTPS termination, database and 
+     other desired supplementary services).
+   - Full Nextcloud stacks, incorporating the image, are provided in the form of example Docker Compose files 
+     and Dockerfile variations. Full feature list available `here <https://github.com/nextcloud/docker/>`_.
+   - `Official Documentation <https://github.com/nextcloud/docker/>`_ (**Note**: see Official Documentation 
+     for Image specific installation and usage details).
+* Nextcloud Web Installer
```

Comment:
Perhaps put a bit more "warnings" that this might or might not work, it depends very much on the shared hosting envioronment. On the forum, there is even a howto with some considerations:
https://help.nextcloud.com/t/how-to-get-help-on-shared-webhosting/178243